src/frontend/src/App.tsx

```diff
@@ -21,15 +25,21 @@ const App: FunctionComponent = () => {
 
   return (
     <Router>
-      <div className={style.app}>
-        <NavBar org={org} user={user} />
-        <div className={style.view}>
-          <Switch>
-            <Route path="/data" component={Data} />
-            <Route path="/" component={Landing} />
-          </Switch>
-        </div>
-      </div>
+      <StylesProvider injectFirst>
```

Comment:
instantiating all the peer deps here. The `injectFirst` prop is needed to prevent css specificity war with CSS in JS library we use `@emotion`